### PR TITLE
Update add on panel status strings

### PR DESCRIPTION
**Pull request by codemist**

### CODE REVIEW

**Review by peiying2 — COMMENTED** [review]
The older version of the string was rewritten based on the i18n guideline. Can week keep the same structure and modify from there.

File: en/messages.json [hunk 1]
```diff
@@ -56,16 +56,15 @@
         "message": "Continue"
     },
     "popupRemainingAliases": {
-        "message": "Remaining aliases: $REMAINING$/$MAX$",
+        "message": "$REMAINING$ aliases remaining.",
```
[the remark this answers is not in the document]
Comment (codemist) [reply en/messages.json]: Ah, noted. Just changed it to `Remaining aliases: $REMAINING$`.

**Review by peiying2 — APPROVED** [review]
Thanks for the fix. Looking good!

File: en/messages.json [hunk 2]
```diff
@@ -56,16 +56,15 @@
...
+        "message": "Remaining aliases: $REMAINING$",
         "description": "Tells the user how many more aliases they can make.",
         "placeholders": {
             "remaining": {
                 "content": "$1"
-            },
-            "max": {
-                "content": "$2"
             }
-        }
```
Comment (mathjazz) [inline en/messages.json:68]: We shouldn't remove that `}`.

**Comment by peiying2** [comment]
@codemist @groovecoder

**Comment by mathjazz** [comment]
I'd suggest adding an automated check to validate JSON and the [schema](https://github.com/mozilla/pontoon/blob/master/pontoon/sync/formats/json_extensions.py#L25-L46) to prevent similar issues in the future.

File: en/messages.json [hunk 3]
```diff
@@ -56,16 +56,15 @@
         "message": "Continue"
     },
     "popupRemainingAliases": {
```
Comment (flodolo) [inline en/messages.json:58]: The ID needs to change when you change the string.

Short explanation: look at [German](https://pontoon.mozilla.org/de/firefox-relay-add-on/all-resources/?search=popupRemainingAliases&string=231315), nobody will know that the text changed, and the add-on will show a `$MAX`.

Long explanation: https://mozilla-l10n.github.io/documentation/localization/making_string_changes.html

Comment (flodolo) [reply en/messages.json:58]: Even better (fresh document which covers both issues in this PR, plurals and string change): https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#choose-self-explanatory-string-ids

**Pull request merged**